yb-voyager/cmd/exportDataDebezium.go

```diff
@@ -246,7 +246,7 @@ func fetchOrRetrieveColToSeqMap(msr *metadb.MigrationStatusRecord, tableList []s
 	case TARGET_DB_EXPORTER_FB_ROLE, TARGET_DB_EXPORTER_FF_ROLE:
 		storedColToSeqMap = msr.TargetColumnToSequenceMapping
 	}
-	if storedColToSeqMap != nil {
+	if storedColToSeqMap != nil && !bool(startClean) {
```

Comment:
why do we need to check startClean again here? It's already been cleaned up, no?

Reply:
yeah its not required but for correctness, I have added as for start clean we don't want to use the MSR one.